app/models/proposal.rb

```diff
@@ -196,6 +196,14 @@ def restart
     Dispatcher.deliver_new_proposal_emails(self)
   end
 
+  def changed_fields comment_text
+    self.comments.create(
+      comment_text: comment_text,
+      update_comment: true,
+      user_id: self.requester_id
+    )
+  end
```

Comment:
I thought this was from a master merge but I don't see the code in master now... will clean this up.